### DISCUSSION TITLE
Save memory db to disk as .db

### DISCUSSION THREAD

**Discussion by dioptre** [duckdb/duckdb#11140] [General]
Now the .db version is becoming more stableish

Can we please work out a way to convert from in memory to .db

Exporting to a directory of files doesn't cut the mustard. 

Thanks!
Andrew

**Comment by Tishj** [comment]
Attach a new persistent database, then use `COPY FROM DATABASE x TO y` <https://duckdb.org/docs/sql/statements/copy#copy-from-database--to>

I think that does what you're looking for?

**Comment by dioptre** [comment]
Ah looks promising but I just stopped using ATTACH / DETACH as it didnt support sequences in my use case https://github.com/duckdb/duckdb-wasm/issues/1520
